docs/Coding/guidelines.rst

```diff
@@ -30,18 +32,21 @@ manually. The downside is that the tool is really picky and will sometimes want
 code. Even the maintainers are sometimes surprised! You will find more information about ``clang-format``
 and detailed instructions in :doc:`/Contributing/pull-request`.
 
-Formatting the code as dictated by ``clang-format`` is mandatory.
+Formatting the code as dictated by ``clang-format`` is :strong:`mandatory`.
 
 
-Copyright Notices
------------------
+Copyright Notice
+----------------
 
 New code follows the `SPDX standard <https://spdx.dev/ids/>`_:
 
 .. code-block:: cpp
 
-    // SPDX-License-Identifier: GPL-3.0-or-later
-    // SPDX-FileCopyrightText: 2022 Author Name <how-to-contact@example.com>
+    /**
+     * SPDX-License-Identifier: GPL-3.0-or-later
+     * SPDX-FileCopyrightText: Freeciv21 and Freeciv Contributors
+     * SPDX-FileCopyrightText: Author Name <how-to-contact@example.com>
+     */
```

Comment:
Aaah no, no `/**` Doxygen here, we don't want this in the docs!

I have a slight preference towards `//`-comments because they're used in Kde, but I've done both and I'd say both are accepted.

Reply:
I was going with the `/*` style for multi-line comments and `//` style for single-line comments. 

> Aaah no, no /** Doxygen here, we don't want this in the docs!

What do you mean here?

Reply:
The SPDX headers should be normal `/*` (or `//`) comments, not `/**` or `///`

Reply:
ok for the record we are all over the map in existing code